libbeat/common/version.go

```diff
@@ -56,6 +56,15 @@ func (v *Version) IsMajor(major int) bool {
 	return major == v.Major
 }
 
+// Equal returns true if v is exactly the same as v1.
+// The meta part is not taken into account.
+func (v *Version) Equal(v1 *Version) bool {
+	if v.Major == v1.Major && v.Minor == v1.Minor && v.Bugfix == v1.Bugfix {
```

Comment:
Would `v.version == v1.version` also work? Or compare `.String()`?

Reply:
I sticked to leaving out the meta information for `rc1` ,`alpha`, `beta` etc as this is also left out in the `LessThan` method. If treeting e.g. `6.0.0.alpha` to not be equal to `6.0.0` then this would lead to the assumption that `6.0.0.alpha` is **less than** `6.0.0` but this returns `false`. 

Will think about how we can change that without breaking existing behavior.